### PR TITLE
Feature formatting bytes suffix

### PR DESCRIPTION
**Pull request by waterfl0w**
This is a PR referencing issue #16. It adds the requested feature,  necessary tests by testing the edge cases, and adjusting the already existing cases to reflect the changes.

### CODE REVIEW

File: jvm-annotations/src/main/java/org/quickperf/jvm/allocation/ByteAllocationMeasureFormatter.java [hunk 1]
```diff
@@ -13,25 +13,42 @@
...
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.text.NumberFormat;
+import java.util.Locale;
+
 public class ByteAllocationMeasureFormatter {
 
     public static final ByteAllocationMeasureFormatter INSTANCE = new ByteAllocationMeasureFormatter();
 
+    private final static DecimalFormat BYTE_PREFIX_FORMATTER = (DecimalFormat) NumberFormat.getInstance(Locale.ENGLISH);
```
Comment (jeanbisutti) [inline jvm-annotations/src/main/java/org/quickperf/jvm/allocation/ByteAllocationMeasureFormatter.java]: From https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormat.html : " It is recommended to create separate format instances for each thread. If multiple threads access a format concurrently, it must be synchronized externally."
Please create on instance for each call

**Comment by jeanbisutti** [comment]
I create a PR on your branch to take in conderiration the comment below and to improve code readability (I hope that you will find the code more readable)... If you have questions, don't hesitate.
If it is ok for you, could you accept my PR, squash the commits in a "Add bytes to allocation display" commit and rebase to master (if you need some help: https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)?
After that, I could merge your PR :)
Many thanks for your work! I hope that you will continue to contribute to QuickPerf :)

**Comment by waterfl0w** [comment]
Thank you for all your help throughout this. It was a really good experience, and I have learned new things along the way. :)

**Comment by jeanbisutti** [comment]
Great! :) You are welcome!

**Comment by jeanbisutti** [comment]
One last thing. There are many commits for this feature. Some unrelated commits (for example "Add Maven wrapper functionality") are in the middle. We would like to keep a Git history easy to read. Could you please, in this specific case, keep the two modified files, refork QuickPerf, add the two modified file in the new fork and submit another PR ? After that I will merge the new PR and add you as contributor in the Readme and pom files. Once again, thanks a lot for your worK.
